### PR TITLE
perf(server): improves data sync performance

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

**Comment by duanyytop** [comment]
CI failed.

**Comment by shaojunda** [comment]
I'am working on it, CI runs on an old version. @duanyytop

File: server/app/models/ckb_sync/data_sync_worker_args_consumer.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module CkbSync
+  class DataSyncWorkerArgsConsumer
+    def initialize(current_worker_args, worker_name, current_round_key)
+      super()
```
Comment (classicalliu) [inline server/app/models/ckb_sync/data_sync_worker_args_consumer.rb]: Can delete this?

Comment (shaojunda) [reply server/app/models/ckb_sync/data_sync_worker_args_consumer.rb]: yes, Thank you.

**Comment by duanyytop** [comment]
CI failed @shaojunda

**Pull request merged**